controller/server/gui.ml

```diff
@@ -253,37 +253,46 @@ module NetworkGui = struct
   open Connman
   open Network
 
+  (** Mutable services resulting from the merge of every
+      services ever given from the overview page.*)
+  let merged_overview_services = ref []
+
+  (** Merge new services into old services.*)
+  let merge_services old_services new_services =
+    old_services
+    |> List.filter
+      (fun (o:Service.t) -> List.exists (fun (n:Service.t) -> o.id == n.id) new_services)
```

Comment:
Is a `not` missing here or am I confused? I assume the intent is to combine previously known services that are no longer found with all services that have now been found.

Reply:
Right, I may have been confused by Ocaml equality testing it.